README.md

```diff
@@ -61,7 +61,7 @@ The [Synchronizer](docs/-synchronizer/README.md) takes care of
     - Downloading the latest compact blocks in a privacy-sensitive way
     - Scanning and trial decrypting those blocks for shielded transactions related to the wallet
     - Processing those related transactions into useful data for the UI
-    - Sending payments to a full node through the light client server
+    - Sending payments to a full node through the light client-server
```

Comment:
Since this is a server for "light clients" we could probably just change this to:
> Sending payments to a full node through [lightwalletd](https://github.com/zcash/lightwalletd)